x-pack/plugins/cases/public/common/navigation/deep_links.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { AppDeepLink } from '../../../../../../src/core/public';
+import { DEFAULT_BASE_PATH, getCreateCasePath, getCasesConfigurePath } from './paths';
+
+export const CasesDeepLinkIds = {
+  cases: 'cases',
+  casesCreate: 'cases_create',
+  casesConfigure: 'cases_configure',
+} as const;
+
+export type CasesDeepLinkId = typeof CasesDeepLinkIds[keyof typeof CasesDeepLinkIds];
```

Comment:
`CasesDeepLinkIds` and `CasesDeepLinkId` have a very similar name (they differ only in one letter). I think it can be very confusing. What about `CasesDeepLinkId` (without ending `s`) -> `ICasesDeepLinks` or similar.